### ISSUE TITLE
helloFromBackup: Hidden comment text is partly omitted

### ISSUE THREAD

**Issue by makise-homura** [stsyn/random-stuff#2]
While using `helloFromBackup.user.js`, I lose any text in hidden comments that isn't wrapped into `<div>`, `<span>`, `<a>` or other blocks. I.e. `<br>Минус за психованную лошадь<br><img src="...">` becomes just `<br><br><img src="...">`. Uh, I'm really a bit embarrassed by Akio's butthurt, but it's not a reason to cut it out at all =)

I guess it's because you construct the comment's content from children nodes (line 107):
```
content: Array.from(sameComment.querySelector('.text').children),
```
while (I guess) it's more confident to put there all `innerHTML` as a whole block:
```
content: sameComment.querySelector('.text').innerHTML,
```
So anything which is not *inside* the children nodes, is lost.

Firefox 84.0, Greasemonkey 4.10.0.

For me, it works if I just replace your implementation by latter one, but you might wish to do it your own way. Or, if you're satisfied with my approach, I can make a PR with it.

**Comment by stsyn** [comment]
Can you please provide a link to the page, where this comment could be founded?

**Comment by makise-homura** [comment]
This one: https://tabun.everypony.ru/blog/news/198655.html#comment13363722

**Comment by stsyn** [comment]
Seems to be browser specific bug, will take a look later
![image](https://user-images.githubusercontent.com/17717421/103287011-405b1380-4a03-11eb-8a56-9d5d8885ccd1.png)

Nevermind, replicated

**Comment by stsyn** [comment]
Fixed in 0.4

**Comment by makise-homura** [comment]
Yeah, looks like working now, thanks!
